### ISSUE TITLE
New link creation UX with Enter key adds '_copy' suffix

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern#2043]
### Expected behaviour

Populating link content text then pressing Enter will create the link.

### Actual behaviour

Populating link content text then pressing Enter creates a suffixed '_copy' of the link.

### Steps to reproduce

Populate a new link title and URL. Press Enter key. Note the '_copy' suffix of the link title.

https://github.com/user-attachments/assets/8afd198f-cf54-49ac-81de-49ff9745a84f

#### Additional information

Textpattern version: 4.9.0

Web server vendor and version: Nginx 1.29.4

Database server vendor and version: Percona Server 8.4

PHP version: 8.5.2

Operating system: Debian 13

**Comment by phiw13** [comment]
Here is a related little nit: on the Presentation > Themes > Edit theme panel, the “duplicate” action is _not_ triggered by pressing enter (with an input field selected). But that “duplicate” action is triggered by a link (`href="#"`). Shouldn’t it be a `button` ? That would have the same issue as described above, though.

That little “hack” of inserting an additional hidden button works, oddly, unclear why. I am not entirely sure it is great accessibility wise (label-less, ‘empty’ and duplicated submit action). Luckily it is or should be invisible.  Amazingly, the Arc accessibility checker does _not_ complain for once, but the Wave checker does and treat that empty button as an error. Both checked in Firefox. The Firefox accessibility inspector does not see that button. As far as I could see (hear…), VO with Safari ignored that button, I think.

**Comment by bloatware** [comment]
That's how browsers react to form submission via 'Enter' key press - they trigger the first (in dom order) submit button click. I wouldn't like to handle it via js, rather altering the visual order via css. Ideas welcome.

**Comment by phiw13** [comment]
> That's fine, but is not its tabbing order logically disturbing? Is there a mean to make 'Save' button's tabindex the last one, but only within the form?

In both cases – the present situation and the eventual substitute – there is visual complexity. And changing that tabbing order as suggested has a precedent: the Save panels for Forms, Pages, Write, … 

> Yes, we just need to choose between anchor and button.

Reverting to using an anchor on the Edit Link panel (TXP 4.8 markup) actually solves the original problem as well, and bypasses the little difficulties in other options.

**Comment by phiw13** [comment]
Ref, option 1,

>  But I am not sure how well and easily that can handled in the Hive stylesheet

Well, that was equally easy as with Sandspace, one line addition to the stylesheet: 

```
.txp-clone { order: -1 }
```

**Comment by Bloke** [comment]
> They are already buttons (save for Themes)

Thanks for unifying those anomalies.
